lua/package-info/config.lua

```diff
@@ -101,7 +110,32 @@ end
 -- @param user_options: M.__DEFAULT_OPTIONS - all the options user can provide in the plugin config
 -- @return nil
 M.__register_user_options = function(user_options)
-    M.options = vim.tbl_deep_extend("keep", user_options or {}, M.__DEFAULT_OPTIONS)
+    if user_options then
+        if user_options.colors and type(user_options.highlights) ~= "table" then
+            logger.warn("`colors` option is deprecated. Use `highlights` instead. Trying to migrate...")
```

Comment:
Maybe also tell the user to check the readme and migrate to the new config and that the old config will be removed.

I can also put a notice in the breaking changes issue. 

What do you think?